src/environments/environment.common.ts

```diff
@@ -212,5 +212,19 @@ export const environment: GlobalConfig = {
   },
   theme: {
     name: 'default',
-  }
+  },
+  curationTasks: [
+    {
+      name: 'profileformats',
+      label: 'curation-task.task.profileformats.label'
+    },
+    {
+      name: 'requiredmetadata',
+      label: 'curation-task.task.requiredmetadata.label'
+    },
+    {
+      name: 'checklinks',
+      label: 'curation-task.task.checklinks.label'
```

Comment:
These configuration seem to duplicate the ones in `curate.cfg`: https://github.com/DSpace/DSpace/blob/main/dspace/config/modules/curate.cfg#L31-L33

That's fine if we need to move these configurations to the UI layer, but then we should remove them from `curate.cfg` on the backend (perhaps in https://github.com/DSpace/DSpace/pull/2820).